meson-scripts/build_libbpf

```diff
@@ -7,18 +7,27 @@ out=${out#\"}
 out=${out%\"}
 args=($out)
 
+declare -a cflags=()
+
 idx=0
 cc=${args[idx]}
 if [ "$cc" = "ccache" ]; then
     idx=$((idx+1))
     cc="$cc ${args[idx]}"
 fi
 
-declare -a cflags=()
+# This fixes an issue that was happening in some cases when building with gcc:
+# relocation R_X86_64_32 against `.rodata' can not be used when making a PIE
+# object; recompile with -fPIE
+if [ "$cc" = "cc" ]; then
```

Comment:
I don't think this is a reliable way to detect gcc. meson has `compiler.has_argument()` test, so maybe test whether `-fPIE` is supported there and add that to build flags?

Reply:
Or let's just always build with `-fPIE` to minimize the build complexity / variance?